tests/test_storage.py

```diff
@@ -651,28 +476,14 @@ def test_upload_prepend_hash(self):
         match = re.match(pattern, blob.name)
         self.assertIsNotNone(match)
 
-    def test_create_bucket(self):
-        """If GCS bucket doesn't exist, create it"""
-        settings = {
-            "storage.bucket": "new_bucket",
-            "storage.region_name": "us-east-1",
-            "storage.gcp_service_account_json_filename": self._config_file,
-        }
-        arguments = GoogleCloudStorage.configure(settings)
-        arguments["bucket_factory"]()
-
-        self.gcs.bucket.assert_called_with("new_bucket")
-        bucket = self.gcs.bucket("new_bucket")
-        bucket.create.assert_called_once_with()
```

Comment:
I removed this test as it is [already](https://github.com/ddelange/pypicloud/blob/210334e0259abe390d532219632188a80ff8aaea/pypicloud/storage/gcs.py#L156) implicitly tested with a fresh `fake-gcs-server` deployment. I assume the test was there to test functionality of the fake GCS classes, which are now removed.